shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java

```diff
@@ -86,6 +93,16 @@ protected void dispatchDraw(Canvas canvas) {
         }
     }
 
+    @Override
+    public void setVisibility(int visibility) {
+        super.setVisibility(visibility);
+        if (visibility == VISIBLE) {
```

Comment:
the way i'm think of, 
There might be people who will do this

```
startShimmerAnimation();
setVisibility(Visible);
```

But I guess this will be user fault? I'm not really sure how would you treat this. so I'll leave it to you to decide on this.